src/Aspire.Dashboard/Components/Controls/ChartContainer.razor.cs

```diff
@@ -170,13 +170,18 @@ protected override async Task OnParametersSetAsync()
 
     private OtlpInstrument? GetInstrument()
     {
+        var endDate = DateTime.UtcNow;
+        // Get more data than is being displayed. Histogram graph uses some historical data to calculate the bucket ranges.
+        // It's ok to get more data than is needed here. An additional date filter applied when building chart values.
```

Comment:
The last sentence feels like something's missing. Should it be:

```suggestion
        // It's ok to get more data than is needed here. An additional date filter is applied when building chart values.
```